### PR TITLE
fix top k of similarity

### PR DESCRIPTION
**Pull request by woodylee1974**
Thank you for contributing to LangChain!

- [ ] **PR title**: "package: description"
  - Where "package" is whichever of langchain, community, core, experimental, etc. is being modified. Use "docs: ..." for purely docs changes, "templates: ..." for template changes, "infra: ..." for CI changes.
  - Example: "community: add foobar LLM"

- [ ] **PR message**: ***Delete this entire checklist*** and replace with
    - **Description:** a description of the change
    - **Issue:** the issue # it fixes, if applicable
    - **Dependencies:** any dependencies required for this change
    - **Twitter handle:** if your PR gets announced, and you'd like a mention, we'll gladly shout you out!

- [ ] **Add tests and docs**: If you're adding a new integration, please include
  1. a test for the integration, preferably unit tests that do not rely on network access,
  2. an example notebook showing its use. It lives in `docs/docs/integrations` directory.

- [ ] **Lint and test**: Run `make format`, `make lint` and `make test` from the root of the package(s) you've modified. See contribution guidelines for more: https://python.langchain.com/docs/contributing/

Additional guidelines:
- Make sure optional dependencies are imported within a function.
- Please do not add dependencies to pyproject.toml files (even optional ones) unless they are required for unit tests.
- Most PRs should not touch more than one package.
- Changes should be backwards compatible.
- If you are adding something to community, do not re-import it in langchain.

If no one reviews your PR within a few days, please @-mention one of baskaryan, efriis, eyurtsev, hwchase17.

### CODE REVIEW

File: libs/community/langchain_community/vectorstores/faiss.py [hunk 1]
```diff
@@ -334,6 +334,7 @@ def similarity_search_with_score_by_vector(
                 for doc, similarity in docs
                 if cmp(similarity, score_threshold)
             ]
+        docs = sorted(docs, key=lambda x: x[1], reverse=True)
```
Comment (baskaryan) [inline libs/community/langchain_community/vectorstores/faiss.py:337]: whether this should be ascending or descending depends on the similarity metric

**Review by baskaryan — COMMENTED** [review]
what's the issue you're seeing?

File: libs/community/pyproject.toml [hunk 2]
```diff
@@ -181,6 +181,9 @@ optional = true
 jupyter = "^1.0.0"
 setuptools = "^67.6.1"
 langchain-core = {path = "../core", develop = true}
+langchain = {path = "../langchain", develop = true}
+langchain-openai = { path = "../partners/openai", develop = true }
+faiss-cpu = "^1.8.0"
```
Comment (Undertone0809) [inline libs/community/pyproject.toml:186]: There's no need to add the dependency in here.

**Review by Undertone0809 — CHANGES_REQUESTED** [review]

**Comment by ccurme** [comment]
Closing as author is unresponsive, can re-open if needed.

**Pull request closed without merging**